src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/partition.py

```diff
@@ -114,9 +114,28 @@ def capture_gptq_patterns(self, graph_module: GraphModule):
                     for pattern_op in enabled_ops:
                         self.supported_ops.enable_by_name(pattern_op)
 
+    def capture_nncf_patterns(self, graph_module: GraphModule):
```

Comment:
may be we should give better naming here because `capture_nncf_patterns` has very general name?

Reply:
I have no ideas. I have agreed with gptq.